### PR TITLE
ChangeLog updates for 1.4.10 and 1.4.11 versions

### PR DESCRIPTION
**Pull request by ichernetsky**
JIRA issues: https://jira.mesosphere.com/browse/MARATHON-7996

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+### Fixed issues
+
+- [MARATHON-7992](https://jira.mesosphere.com/browse/MARATHON-7992) - Validation errors on root group in large clusters can lead Marathon to crash.
+
+
+## Changes from 1.4.9 to 1.4.10
+
+### Fixed issues
+
+- [MARATHON-7974](https://jira.mesosphere.com/browse/MARATHON-7974) - Fixes a security concern for a non super user. It was discovered that permissions for GroupResources was not honored and it is now resolved.
```
Comment (timcharper) [inline changelog.md]: I think that this is more alarming than it needs to be. It wasn't that we have a security vulnerability, but rather, only super users could access things. So, it was the opposite of a vulnerability (security was too tight).

Comment (timcharper) [reply changelog.md]: @kensipe can you please confirm that my understanding here is correct?

Comment (ichernetsky) [reply changelog.md]: That's how it is phrased on the releases page: https://github.com/mesosphere/marathon/releases
Should it be rephrased anyway?

Comment (timcharper) [reply changelog.md]: Yes, I think it should be. Also, I think we should fix the release notes.

File: changelog.md [hunk 2]
```diff
@@ -1,6 +1,24 @@
+## Changes from 1.4.10 to 1.4.11
+
+### Fixed issues
+
+- [MARATHON-7992](https://jira.mesosphere.com/browse/MARATHON-7992) - Validation errors on root group in large clusters can lead Marathon to crash.
```
Comment (kensipe) [inline changelog.md]: I'm looking at https://github.com/mesosphere/marathon/compare/v1.4.10...releases/1.4

I see MARATHON-7990 but not MARATHON-7992

Comment (ichernetsky) [reply changelog.md]: It is for 1.4.11

Comment (kensipe) [reply changelog.md]: looks like slack is down :( .   The issue is in the tracking (and not on your end).   Marathon-7992 is Unassigned and in review however it has #5874 in the comments.   #5874 is merged (so not in review) and lists MARATHON-7990 as it's JIRA issue.

I'll approve but under duress :)

**Comment by ichernetsky** [comment]
https://jira.mesosphere.com/browse/MARATHON-7992 is the correct one. It looks like the one was just copy and pasted.

**Comment by ichernetsky** [comment]
*the other on* was copy and pasted.

File: changelog.md (hunk 2 above)
Comment (timcharper) [reply changelog.md]: Yeah, my bad. At least I added a PR link in the comments, though? :)

Comment (ichernetsky) [reply changelog.md]: It is a tiny thing, not even worth mentioning. Merging :)

**Pull request merged**